.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,49 @@
+---
+name  : Bug report
+about : Report a bug in Shiny.
+---
+
+<!--
+
+Before you file an issue, we would appreciate it if you would upgrade to the latest version of Shiny from CRAN and confirm that the problem persists.
+
+# To install latest shiny from CRAN:
+install.packages("shiny")
+# To update all your R packages:
+update.packages(ask = FALSE, checkBuilt = TRUE)
+# Please remember to restart R after updating packages.
+
+This issue tracker is for bugs and feature requests in the Shiny package. If you're having trouble with Shiny Server or a related package, please file an issue in the appropriate repository. 
+
+If you're having trouble with shinyapps.io, and you have a paid account (Starter, Basic, Standard, or Pro), please file a support ticket via https://support.rstudio.com. If you have a Free account, please post to the RStudio Community with the shinyappsio tag: https://community.rstudio.com/tags/shinyappsio.
+
+Finally, if you are an RStudio customer and are having trouble with one of our Pro products, get in touch with our support team at support@rstudio.com.
+
+See our guide to writing good bug reports for further guidance: https://github.com/rstudio/shiny/wiki/Writing-Good-Bug-Reports. The better your report is, the likelier we are to be able to reproduce and ultimately solve it.
+-->
+
+### System details
+
+    Shiny Version           :
```

Comment:
I think it's better to have people call `sessionInfo()` and paste it here. It's both more informative and easier to do. (I know this because when I report bugs on the RStudio IDE, I get annoyed at having to look up versions of everything and enter them in.)

I also think we shouldn't have people fill out Browser Version and RStudio Version -- those are extra hurdles that are relevant in a relatively small proportion of cases.